src/lib/connect/ws.ts

```diff
@@ -1,12 +1,15 @@
 import { StreamBuilder } from '../shared'
-
 import { Buffer } from 'buffer'
-import WS, { ClientOptions } from 'ws'
+import WebSocket, {
+	ClientOptions,
+	ErrorEvent,
+	MessageEvent,
+} from 'isomorphic-ws'
```

Comment:
This is a nice package that automatically uses ws on node and global.Websocket on browser. This makes types more clear